_data/fr/navigation.yml

```diff
@@ -0,0 +1,28 @@
+nav:
+- title:                 Utilisation
+  subfolderitems:
+    - page:              Démarrage et installation
+      url:               /wiki/Getting-Started
+      subsubfolderitems:
+        - page:          Windows
+          url:           /wiki/Installation-for-Windows
+        - page:          Macintosh
+          url:           /wiki/Installation-for-Macintosh
+        - page:          Linux
+          url:           /wiki/Installation-for-Linux
+    - page:              Mode d'emploi
+      url:               /wiki/Software-Manual
+    - page:              Dépannage
+      url:               /wiki/Client-Troubleshooting
+- title:                 Aller plus loin…
+  subfolderitems:
+    - page:              Gérer un serveur Jamulus
+      url:               /wiki/Running-a-Server
+    - page:              Configuration et options
+      url:               /wiki/Command-Line-Options
+    - page:              Trucs et astuces
+      url:               /wiki/Tips-Tricks-More
+    - page:              Démonstrations
```

Comment:
Could be "Démos" as its English counterpart.

Reply:
Je continue en français, si ça ne pose pas de problèmes. (I hope!)
@Samyx a, dans son travail, fait la même traduction que moi : « Démonstrations », mais « Exemples » ou « Exemple d'utilisation » serait beaucoup plus clair et et cohérent avec le contenu de la section.